components/compliance-service/integration_test/reports_projects_update_test.go

```diff
@@ -410,6 +410,30 @@ func TestProjectUpdate(t *testing.T) {
 			},
 			projectIDs: []string{},
 		},
+		{
```

Comment:
These two tests failed with the same elasticsearch error from the issue before the above changes.

Reply:
Not sure if this should be a refactor or an additional test but it'd be great to see a test struct with all possible combinations of `attribute`, `operator`, and `value` (array or single value), to make sure we're not missing any edge cases. We could use two fully-populated `&relaxting.ESInSpecSummary` structs, one expected to match and one not to match, and iterate over the condition combinations on each test run.

Reply:
I would like to cover all the cases. I don't fully understand what you are proposing, but it sounds interesting. 
The reason you don't see the `operator` in these tests is because it is completely ignored in the code.